.jenkins/cscs/env-clang-cuda.sh

```diff
@@ -12,23 +12,24 @@ export CXX_STD="17"
 export HWLOC_ROOT="${APPS_ROOT}/hwloc-2.0.3-gcc-8.3.0"
 
 module load daint-gpu
-module load cudatoolkit/10.2.89_3.29-7.0.2.1_3.5__g67354b4
+module load cudatoolkit/11.0.2_3.33-7.0.2.1_3.1__g1ba0366
 module load Boost/1.75.0-CrayCCE-20.11
-spack load cmake@3.17.3
+spack load cmake@3.18.6
 spack load ninja@1.10.0
 
 export CXX=`which CC`
 export CC=`which cc`
 
 configure_extra_options="-DCMAKE_BUILD_TYPE=Debug"
 configure_extra_options+=" -DHPX_WITH_CUDA=ON"
-configure_extra_options+=" -DHPX_WITH_CUDA_CLANG=ON"
-configure_extra_options+=" -DHPX_CUDA_CLANG_FLAGS=\"--cuda-gpu-arch=sm_60\""
+configure_extra_options+=" -DCMAKE_CUDA_FLAGS=\"--cuda-gpu-arch=sm_60\""
```

Comment:
Watch out for the `CMAKE_CUDA_FLAGS` for the Boost workaround. They need to be combined into one.

Reply:
There are still two `CMAKE_CUDA_FLAGS`. One here and one for the boost workaround.